samples/client/petstore/graphql-client/petstore/api/pet_api.graphql

```diff
@@ -0,0 +1,60 @@
+# OpenAPI Petstore
+# This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+# OpenAPI spec version: 1.0.0
+# Generated by OpenAPI Generator: https://openapi-generator.tech
+#
+
+# package petstore
+
+# pet_api
+
+type Query {
+  # Add a new pet to the store
+  # @param Pet $pet Pet object that needs to be added to the store
+  # @return [null]
+  AddPet($pet: Pet!): null
+
+  # Deletes a pet
+  # @param Int $petId Pet id to delete
+  # @param String $apiKey 
+  # @return [null]
+  DeletePet($petId: Int!, $apiKey: String): null
```

Comment:
this should go in a mutation

Reply:
Yes.
this one, AddPet, UpdatePet, and UpdatePetWithForm queries should be in mutation
a good rule of thumb is that "GET & HEAD" verbs in HTTP are graphql queries and every other verb should be a mutation

Reply:
Thanks for the feedback. I've fixed it via https://github.com/OpenAPITools/openapi-generator/pull/609/commits/6dcf74e700fc3adadd66c9cd93453825b554a303